src/com/vividsolutions/jump/workbench/ui/WorkbenchFrame.java

```diff
@@ -633,11 +632,7 @@ public void log(String message, Class clazz) {
      */
     @Deprecated
     public void log(String message, Throwable t) {
-        log(message, t, new Exception().getStackTrace()[0]);
-    }
-
-    private void log(String message, Throwable t, StackTraceElement calledFrom) {
-        Logger.log(message, Logger.isDebugEnabled() ? t : null, Level.INFO, calledFrom);
+        Logger.info(message, t);
```

Comment:
same here. let's expose Log-Levels via static Members of class Logger to get rid of the `org.apache.log4j.Level` import instead.